.all-contributorsrc

```diff
@@ -102,6 +102,13 @@
         "code",
         "test"
       ]
+    },
+    {
+      "login": "amilajack",
+      "name": "Amila Welihinda",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/6374832?v=3",
+      "profile": "http://amilajack.com",
+      "contributions": []
```

Comment:
Could you add "infra" to this (because that's pretty much what [your PR was about](https://github.com/kentcdodds/cross-env/pull/26/files)). Then you'll want to regenerate the contributors table via: `npm start generateContributors`